localization/english/country_flavor_text_l_english.yml

```diff
@@ -52,4 +52,6 @@
  NNG_FLAVOR_TEXT:0 "Hongshanman, or red barbarians from the mountains, is the name the Chinese settlers gave to the aboriginal tribes in the continent after initial contact. Over time, those friendly with the Aboriginals would drop the -man, or barbarian, and they would become just the Hongshan."
  YMT_FLAVOR_TEXT:0 "Hongshanman, or red barbarians from the mountains, is the name the Chinese settlers gave to the aboriginal tribes in the continent after initial contact. Over time, those friendly with the Aboriginals would drop the -man, or barbarian, and they would become just the Hongshan."
  KOO_FLAVOR_TEXT:0 "Hongshanman, or red barbarians from the mountains, is the name the Chinese settlers gave to the aboriginal tribes in the continent after initial contact. Over time, those friendly with the Aboriginals would drop the -man, or barbarian, and they would become just the Hongshan."
-
+ OPS_FLAVOR_TEXT:0 "Forced out during the Winkelier uprising, the Hoofkommandant and his supporters retreated to Katzia, here they plot their return to Niew Antwerpen to restore order over Nuweflander."
```

Comment:
Might want to provice context on what the Winkelier uprising is, for instance(note I don't know what it is) "Forced out from 'Nuweflander' during the Winkelier uprising, the Hoofkommandant and his supporters retreated to Katzia, where they plot their return to Niew Antwerpen to restore order over Nuweflander."

Reply:
yeah i might leave it out of the flavour text for the country but im going to put context in the opening event fro nuweflander and opstalia. 
fair bit of lore here, basically just added some new names for flavour into the already existing lore.
as nuweflander is the only part of the cape that produces commodoties like coffee etc. the plantation owners here became the wealthiest geuzen in the cape. this is a big reason why they were the first to rebel from Burgundy as the tax hikes during the german wars hit them the hardest. the rebellion was led by Hendrik van Boetzelaer, a plantation owner, who rather than transition into a republic established the Grondraad (land(s) council) with his fellow plantation owners to maintain power as an oligarchy. The merchants, shopkeepers and other petty bourgeoise in Niew Antwerpen (known as Winkeliers, translated from shopkeeper/ tradesman) didn't like a government that favoured wealthy landowners and forced hendrik out (winkelier uprising) and established a republic